resources/css/app.css

```
@@ -57,6 +57,8 @@
     --color-sidebar-accent-foreground: var(--sidebar-accent-foreground);
     --color-sidebar-border: var(--sidebar-border);
     --color-sidebar-ring: var(--sidebar-ring);
+
+    --color-footer-background: var(--footer-background);
```

Should be named just --color-footer and changed based on my other comments.
```suggestion
    --color-footer: var(--footer);
    --color-footer-foreground: var(--footer-foreground);
```